### PR TITLE
Story/tg 376

### PR DESCRIPTION
**Pull request by Hliebov**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/202" target="_blank">ReviewNinja

File: src/app/constructor/devices/devices.template.html [hunk 1]
```diff
@@ -2,10 +2,19 @@
...
+        <div class="device__icon"
+             [style.background-image]="'url(assets/' + device.mqttId + '.svg)'">
+        </div>
+        <div *ngIf="device.type === 'switcher'"
+             (click)="switchSensor(device)">
+            <div class="device__clickable"
+                 *ngIf="device.executor"
+                 [style.background-image]="'url(assets/switcherON.svg)'">
+            </div>
+            <div class="device__clickable"
```
Comment (piton13) [inline src/app/constructor/devices/devices.template.html]: !fix 
Please use only one div and move get url logic into component to test it.

Comment (Hliebov) [inline src/app/constructor/devices/devices.template.html]: Thanks! Done

File: src/app/constructor/devices/devices.component.ts [hunk 2]
```diff
@@ -28,6 +28,10 @@ export class DevicesComponent implements AfterViewInit {
         device.executor = !device.executor;
     }
 
+    switcherImageUrl(device: Device): string {
```
Comment (piton13) [inline src/app/constructor/devices/devices.component.ts:31]: What about tests for it? :)

File: src/app/constructor/devices/devices.template.html (hunk 1 above)
Comment (piton13) [inline src/app/constructor/devices/devices.template.html]: !fixed

**Pull request merged**